src/components/Products/ProductImage.js

```diff
@@ -19,12 +19,24 @@ class ProductImage extends Component {
     };
     
     var isThereAFile = (product) => {
-      fileId = this.props.product.relationships.files.data[0].id;
-      file = this.props.products.included.files.find(function (el) {
-        return fileId === el.id
-      })      
-        return <img alt={this.props.product.name + ', ' + this.props.product.description} src={file.link.href} style={{"background": this.props.background}}/> || <img alt="placeholder" src={placeholder}/>
-    };
+      
+      if(this.props.product.relationships.files) {
+        
+        fileId = this.props.product.relationships.files.data[0].id;
+        
+        file = this.props.products.included.files.find(function (el) {
+          return fileId === el.id
+        })    
+          
+          return <img alt={this.props.product.name + ', ' + this.props.product.description} src={file.link.href} style={{"background": this.props.background}}/>
```

Comment:
I can see [instances](https://github.com/moltin/react-demo-store/blob/2b25d145ef472d6effa22e2af84e4153a891d5ca/src/components/Products/AllProducts.js#L36) where the `ProductImage` component is invoked without a `background` property. What happens in those cases?